generators/server/templates/src/main/resources/config/_application-prod.yml

```diff
@@ -25,6 +25,8 @@ eureka:
 <%_ } _%>
 
 spring:
+    profiles:
+        active: prod
```

Comment:
can we remove this as its not required, this will work from the `application.yml` itself